frontend/src/components/user/Header.tsx

```diff
@@ -19,67 +18,38 @@ type Props = {
 }
 
 const Header: React.FC<Props> = ({ user, actions: { logout } }): JSX.Element => {
-  const router = useRouter()
-  const [isOpen, setIsOpen] = useState(false)
-
+  const Logo = '/img/logo.png'
   return (
-    <header className={classNames(
-      'bg-white border-b shadow-sm',
-      router.pathname.includes('/categories/questions') ?
-      'cursor-not-allowed opacity-50' : ''
-    )}>
-      <nav className={classNames(
-        'max-w-4xl mx-auto px-4 py-4 flex-wrap',
-        'flex items-center justify-between'
-      )}>
-        <div>
-          <h1 className={classNames(
-            'font-semibold text-lg text-gray-900'
-          )}>Sun<span className="text-orange-500">*</span> ELearning App</h1>
-        </div>
-        <div className="block md:hidden">
-          <button
-            onClick={() => setIsOpen(!isOpen)}
-            className="p-1.5 rounded-full hover:bg-orange-50 focus:bg-orange-100"
-          >
-            <BiMenuAltRight className="w-7 h-7 fill-current" />
-          </button>
-          <DropDownMenuToggle 
-            isOpen={isOpen} 
-            logout={logout} 
-            setIsOpen={setIsOpen} 
-          />
+    <header className="bg-white py-5 px-5 shadow-primary transition-all duration-500">
+      <div className="flex items-center justify-between max-w-6xl mx-auto">
+        <div className="flex items-center">
+          {/* Logo */}
+          <Link href="/dashboard">
+            {/* eslint-disable-next-line @next/next/no-img-element */}
```

Comment:
remove comment if not needed

Reply:
I cannot removed that It will get an error for my eslint typescript